### ISSUE TITLE
ResNet Code

### ISSUE THREAD

**Issue by DQDH** [speedinghzl/DSRG#3]
Hi, @speedinghzl . Can you release the code for resnet101 result in the paper? Thanks.

**Comment by speedinghzl** [comment]
@hardBird123 Sorry about the late reply. I have no plan to release the ResNet code. Thanks.
It is not difficult to achieve good performance with Resnet-101,  in fact, overfitting is a key obstacle. 
I just added a dropout layer with ratio 0.5 before the last layer and adjusted the learning rate to avoid gradient explosion.

**Comment by DQDH** [comment]
@speedinghzl  Thanks. I have another problem that when I running the code, it is divided two parts, one always in GPU0 ,another part in GPU2, can you tell how to modify the code？

**Comment by speedinghzl** [comment]
@hardBird123 You can add **CUDA_VISIBLE_DEVICES=2** before the bash command in run.sh. At the same time, setting GPU=0 in run.sh.

**Comment by DQDH** [comment]
Thanks a lot!
